Python/Product/PythonTools/PythonTools/Debugger/DebugAdapter/DebugInfo.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Microsoft.PythonTools.Debugger {
+
+    [JsonObject(MemberSerialization.OptIn)]
+    public abstract class DebugInfo {
```

Comment:
you should be able to make these classes internal

Reply:
Will be updated in next commit